stoqdrivers/printers/bematech/MP4000.py

```diff
@@ -23,8 +23,8 @@
 log = logging.getLogger('stoqdrivers.bematech.MP4000')
 _ = stoqdrivers_gettext
 
-# CMD_ADD_ITEM = 0x3e47 # this is different from mp25
-CMD_ADD_ITEM = 0x09  # Simple add item
+CMD_ADD_ITEM = 0x3e47  # this is different from mp25
```

Comment:
@suniagajose did you test this? because we had some issues with some commands and the panama printer